File: 1.5-ArrayBasics/ArrayBasics_in_Python/1-LinearSearch.py
def linearSearch(arr,s):

    if s in arr:
        return True
    else:
        return False
    

arr = list(map(int, input('Enter 5 element in array').split()))

s = int(input("element to be searched ? "))

ans = linearSearch(arr, s)

if ans:
    print("present in array")
else:
    print("not present in array")

File: 1.5-ArrayBasics/ArrayBasics_in_CPP/1-LinearSearch.cpp
#include<bits/stdc++.h>
using namespace std;


bool linearSearch(int arr[], int n, int s){

    for(int i=0; i<n; i++){
        if(arr[i]==s){
            return true;
        }
    }

    return false;

}

int main(){

    int arr[5];

    int n = sizeof(arr)/sizeof(int);

    cout<<"Enter 5 Elements in array : "<<endl;

    // input in array
    for(int i=0; i<n; i++){
        cin>>arr[i];
    }

    cout<<"Element to Search : "<<endl;
    int s;
    cin>>s;

    int res = linearSearch(arr, n, s);

    if(res){
        cout<<"present in array";
    }
    else{
        cout<<"not present in array";
    }

    return 0;
}